cms/functions.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/* Doc Block
+ * Returns all project titles from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project titles in the arrays within the arrays pulled from the database.
+ */
+function returnProjectTitles(array $array):string {
+    $titles = '';
+    foreach ($array as $value) {
+        $titles .=$value['projectTitle']."\n";
+    }
+    return $titles;
+}
+
+/* Doc Block
+ * Returns all project title links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project title links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectTitleLinks(array $array):string {
+    $titlesLinks = '';
+    foreach ($array as $value) {
+        $titlesLinks .= $value['projectTitleLink']."\n";
+    }
+    return $titlesLinks;
+}
+
+/* Doc Block
+ * Returns all project image links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project image links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectImages(array $array):string {
+    $projectImages = '';
+    foreach ($array as $value) {
+        $projectImages .= $value['projectImage']."\n";
+    }
+    return $projectImages;
+}
+
+/* Doc Block
+ * Returns project id's from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of id's or keys in the arrays within the arrays pulled from the database.
+ */
+function returnProjectIds(array $array):string {
```

Comment:
duplication again. can be done programatically in function above

Reply:
Added one function for whole form with all default values.